var/spack/repos/builtin/packages/texlive/package.py

```diff
@@ -29,8 +29,8 @@ class Texlive(AutotoolsPackage):
     # connection at install time and the package versions could change over
     # time. It is better to use a version built from tarballs, as defined with
     # the "releases" below.
-    version('live', sha256='7c90a50e55533d57170cbc7c0370a010019946eb18570282948e1af6f809382d',
-            url='ftp://tug.org/historic/systems/texlive/2020/install-tl-unx.tar.gz')
+    version('live', sha256='74eac0855e1e40c8db4f28b24ef354bd7263c1f76031bdc02b52156b572b7a1d',
```

Comment:
I am getting the following sha256 using `spack checksum textlive@live`:

```suggestion
    version('live', sha256='7c90a50e55533d57170cbc7c0370a010019946eb18570282948e1af6f809382d',
```
How did you get the value you're using above?